src/Compiler/CompilerDefines.cpp

```diff
@@ -140,3 +142,7 @@ bool FOEDAG::target_device(const QString &target) {
                            qPrintable(QString("target_device %1").arg(target)));
   return (res == TCL_OK);
 }
+
+bool FOEDAG::IsEqual(const QString &str, const QString &s) {
```

Comment:
This sounds like a place for the utils class, no ?

Reply:
Done

Reply:
Why not the string utils ? How are we planning to differentiate between QtUtils and StringUtils ?
https://github.com/os-fpga/FOEDAG/blob/main/src/Utils/StringUtils.cpp

Reply:
StringUtils related to std::string. If we have something that related to Qt it should go to QtUtils